Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Models/Products.cs

```diff
@@ -7,14 +7,22 @@ namespace Know_Your_Nation_Speedy.Models
 {
     public class Products
     {
+        public Products()
+        {
+            Initialise();
+        }
         public int ProductsId { get; set; }
         public string Name { get; set; }
         public string CoverImageLocation { get; set; }
         public string Description { get; set; }
         public int Rating { get; set; }
         public double Price { get; set; }
-        public string Type { get; set; }//Know your nation or Speedy
+        public string Type { get; set; }
         public int QuantityOnHand { get; set; }
-        public ICollection<ProductOrders> ProductOrder { get; set; }
+        public HashSet<ProductOrders> ProductOrder { get; set; }
```

Comment:
this should be an icollection

Reply:
so the idea is to keep it as an Icollection, but instantiate it as a HashSet?

Reply:
would i need to create a new migration if i change this to Icollection?